module/VuFind/src/VuFind/Controller/CoverController.php

```diff
@@ -129,10 +129,13 @@ public function showAction()
         if (!empty($url)) {
             try {
                 $image = $this->proxy->fetch($url);
-                return $this->displayImage(
-                    $image->getHeaders()->get('content-type')->getFieldValue(),
-                    $image->getContent()
-                );
+                $contentType = $image->getHeaders()->get('content-type')->getFieldValue();
```

Comment:
This should use optional chaining, since the header could be missing (an old problem, it seems).